### PR TITLE
Corrected BFS example description in user guide

### PR DESCRIPTION
**Pull request by rekhajoshm**
Corrected BFS example description in user guide
- Corrected Bob ->Esther/age description
- Corrected package and paper link
- Retained Belief propagation where it was relevant, removed duplicate.
- Removed unnecessary import

### CODE REVIEW

**Comment by felixcheung** [comment]
thanks, anything else you can see that needs to be fixed?

**Comment by rekhajoshm** [comment]
updated for few more corrections. thanks @felixcheung

File: docs/user-guide.md [hunk 1]
```diff
@@ -872,9 +873,6 @@ val agg = { g.aggregateMessages
 agg.show()
 {% endhighlight %}
 
-For a more complex example, look at the code used to implement the
-[Belief Propagation example](api/scala/index.html#org.graphframes.examples.BeliefPropagation$).
-
```
Comment (felixcheung) [inline docs/user-guide.md]: why is this removed?

**Comment by rekhajoshm** [comment]
Thanks @felixcheung  
imo, the value methods in the BeliefPropagation are relevant to the other document section(GraphX/GraphFrames graphical models) where Belief Propagation link is retained.
``` def runBPwithGraphFrames(g: GraphFrame, numIter: Int): GraphFrame
def runBPwithGraphX(g: GraphFrame, numIter: Int): GraphFrame ```
The duplicate reference however is removed as bit confusing, it did not help me to cleanly make connection from the document section on aggregates to the link itself. 
Please let me know what you think. Thanks.

**Comment by felixcheung** [comment]
I thought both cases make sense as examples since both of them are implemented with aggregateMessages?

**Comment by rekhajoshm** [comment]
updated.Please verify @felixcheung

**Comment by rekhajoshm** [comment]
@felixcheung can this be merged? thanks.

**Comment by felixcheung** [comment]
thx!
